### PR TITLE
Fixing docs typos

### PR DESCRIPTION
**Pull request by mmerce**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -298,7 +313,7 @@ for).
     5.8,2.8,5.1,2.4,Iris-virginica
 
 If your credentials are stored in the environment as mentioned above,
-you can easily generate a prediction following these steps:
+you can easily generate a prediction following these steps::
 
     $api = new BigML\BigML();
```
Comment (sdesimone) [inline README.rst]: No `code-block:: php`? (also applies to all other instances of code not tagged)

Comment (mmerce) [reply README.rst]: you can either use the :: or the code-block. The problem was that some code snippets had only : and the code was not correctly displayed.

**Comment by sdesimone** [comment]
> On 04 Jun 2018, at 18:21, mmerce <notifications@github.com> wrote:
> 
> @mmerce commented on this pull request.
> 
> In README.rst <https://github.com/bigmlcom/bigml-php/pull/35#discussion_r192801262>:
> 
> > @@ -298,7 +313,7 @@ for).
>      5.8,2.8,5.1,2.4,Iris-virginica
> 
>  If your credentials are stored in the environment as mentioned above,
> -you can easily generate a prediction following these steps:
> +you can easily generate a prediction following these steps::
> 
>      $api = new BigML\BigML();
> you can either use the :: or the code-block. The problem was that some code snippets had only : and the code was not correctly displayed.
> 

great! this is maybe no important at all, but I guess using `::` will not give you syntax coloring (since `code-block` also include a language spec).
if that is right and we are fine with it, no problem on my side.

**Comment by mmerce** [comment]
I can change every code snippet to code-block, if that's what you mean. The present change only fixes the problem showing the snippets that had no formatting.

**Review by sdesimone — COMMENTED** [review]
looks great!

**Review by sdesimone — APPROVED** [review]

**Pull request merged**